db/migrate/20240117180824_latin1_to_utf8.rb

```diff
@@ -0,0 +1,15 @@
+class Latin1ToUtf8 < ActiveRecord::Migration[7.1]
+  def up
+    Article.connection.
+      execute("ALTER TABLE articles CONVERT TO CHARACTER SET utf8mb4;")
+    Sequence.connection.
+      execute("ALTER TABLE sequences CONVERT TO CHARACTER SET utf8mb4;")
+  end
+
+  def down
+    Article.connection.
+      execute("ALTER TABLE articles CONVERT TO CHARACTER SET latin1;")
+    Sequence.connection.
+      execute("ALTER TABLE sequences CONVERT TO CHARACTER SET latin1;")
+  end
```

Comment:
I should probably delete the down migration. It was OK for testing purposes.
But it will blow up if either of these tables has a single non-latin character.

Reply:
Sounds right to me.